mediathread/main/views.py

```diff
@@ -723,6 +733,30 @@ class HomepageView(LoggedInAsFacultyMixin, CourseListView):
 
     def get_context_data(self, **kwargs):
         context = super(HomepageView, self).get_context_data(**kwargs)
-        activatable_courses = []
+        if not waffle.flag_is_active(self.request, 'instructor_homepage'):
+            return context
+
+        groups = self.request.user.groups.all()
```

Comment:
Picking up all groups will also include non-course based groups associated with collaborations. There's some complexity here as well as to what groups are created for the user at login. Maybe we can walk through this flow together?